corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -300,6 +300,7 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
             this.sortIndex = null;
             this.search = null;
             this.queryData = null;
+            this.sessionId = null;
```

Comment:
Should this also be cleared if the user navigates away from the form? I'd think the places where `selectedValues` are removed from session would be the same places the form session id would be cleared.